developer-docs-site/docs/concepts/gas-txn-fee.md

```diff
@@ -133,7 +133,7 @@ We are aware of the confusion this might create, and plan to present these as se
 
 If a transaction deletes state items, a refund is issued to the transaction payer for the released storage slots. Currently, a full refund is issued for the slot's fee, excluding any fees for excess bytes beyond a set quota (e.g., 1KB). However, fees for event emissions are not refundable.
 
-The refund amount is denominated in APT and is not converted to gas units or included in the total `gas_used`. Instead, this refund amount is specifically detailed in the `storage_fee_refund_octas` field of the [`FeeStatement`](#the-fee-statement). As a result, the transaction's net effect on the payer's APT balance is determined by `gas_used * gas_unit_price - storage_refund`. If the result is positive, there is a deduction from the account balance; if negative, there is a deposit. 
```

Comment:
nit: placing each sentence on each line helps a lot with makings diffs smaller and leaving comments

Reply:
Well this is markdown, so if you put things are a new line, they'll be rendered as a separate paragraph.

Reply:
I am pretty sure you can place them on the next line, because the space is used for new paragraphs. Anyway,  this is my personal preference because it is way easier to refer to a specific sentence this way :)

Reply:
Yeah I deleted my previous comment, realizing I was confused by myself.

I'll clean up the new doc I wrote and split things into lines, but probably won't fix other docs at this moment.